sycl/test/basic_tests/buffer/buffer_interop.cpp

```diff
@@ -183,5 +182,86 @@ int main() {
     CHECK_OCL_CODE(clReleaseEvent(OpenCLEvent));
   }
 
+  {
+    queue myQueue;
```

Comment:
```suggestion
    queue MyQueue;
```

Reply:
Is it still popular code style to use `my`/`My` prefix for variables names?

Reply:
It's hard to say. `q`, `Queue`, and `TestQueue` are also popular.

Reply:
> It's hard to say. `q`, `Queue`, and `TestQueue` are also popular.

IMO, any of these are better than `MyQueue`.

Reply:
I only moved the code between tests without looking at it.  Will do the renaming to align with the surrounding code.